test/cpp/jit/test_gpu.cpp

```diff
@@ -103,14 +109,28 @@ bool isPredicated(TensorView* tv, GpuLower& gpulw) {
 
 } // namespace
 
+class NVFuserTest: public ::testing::Test {
+ protected:
+  void SetUp() override {
+    skipOnPreVoltaGPU();
+  }
+
+  void skipOnPreVoltaGPU() {
+    if (at::cuda::getDeviceProperties(0)->major < 6) {
+      GTEST_SKIP() << "Volta or newer GPU is required";
+      return;
+    }
+  }
+};
```

Comment:
Why don't we move this to a common header filer for our C++ tests?

Reply:
already did this :stuck_out_tongue: